### PR TITLE
setup yarn (fixes #97)

### LINKED ISSUE #97: Migrate to yarn

### PR DESCRIPTION
**Pull request by just-boris**
Build time before: ~5min
Build time after: ~5min

### CODE REVIEW

**Comment by just-boris** [comment]
To be precise, I have run build 3 times. Result is better with every attempt:
https://ci.qameta.in/job/allure2_pull-request/139/ – 5 min 28 sec
https://ci.qameta.in/job/allure2_pull-request/141/ – 4 min 49 sec
https://ci.qameta.in/job/allure2_pull-request/142/ – 4 min 30 sec

**Comment by baev** [comment]
@just-boris thanks!

File: allure-web/.gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ coverage
 etc
 node_modules
 node
+yarn.lock
```
Comment (lanwen) [inline allure-web/.gitignore]: you should commit this file as mentioned in https://yarnpkg.com/en/docs/yarn-lock#toc-check-into-source-control

Comment (just-boris) [inline allure-web/.gitignore]: I have already tried that in other projects. It causes merge conflicts for no reason. This problem is bigger than benefits from fixed package versions

**Pull request merged**